integrationTests/extraResourceBases/src/main/java/org/akhikhl/examples/gretty/hellogretty/ExampleServlet.java

```diff
@@ -10,10 +10,10 @@
 
 import java.io.IOException;
 import java.io.PrintWriter;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import javax.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
```

Comment:
I guess if we follow the style for sorted imports, these should go above `java`, but Gretty is anyway full of style errors so it's not that important. :)

Reply:
I accept that criticism because the whole commit is really _only_ about imports. Then they should be done proper ;) And it took me only 2 mins to figure things out in IntelliJ. Please tell me if the fixup commit fits.